### ISSUE TITLE
Plugin commands don't work

### ISSUE THREAD

**Issue by poustmal** [Hydreon/Steadfast2#610]
I have edited some plugins for api 1.12.0 and commands don't work on some plugins when i have joined.I opped myself and I have again the same problem.What sould I do?Or can you fix it?

**Comment by TobiasDevDE** [comment]
Are there any errors etc occuring?

**Comment by poustmal** [comment]
No errors,just when you join the server with 1.1 version,server says unknown command when you type some commands from some plugins that are installed.

**Comment by HerryYT** [comment]
are you sure that on plugin.yml they are registered? or even you registered events? i usually make external classes for commands and after i register it. (for stead i reccomand it).

**Comment by TobiasDevDE** [comment]
Yes, it is an common fault that you forget to register Commands and / or Events. I recommend you to check your code and then give us feedback.
